backend/api/models.py

```diff
@@ -50,7 +50,23 @@ class FullCourseBuildSettings(BaseCourseBuildSettings):
     course_enrollment_key: Optional[str] = None
     academic_year: str
     academic_year_short: str
-    creator: str
+    creator_email: str
+    status: Literal["created", "processing", "completed", "failed"]
+
+
+class GetCourseBuildSettings(BaseModel):
```

Comment:
I saw your comment [here](https://github.com/openstax/rope/pull/16#issuecomment-1971630453), but I'm still confused as to why this is needed. Specifically, it looks like the only difference between `GetCourseBuildSettings` and `FullCourseBuildSettings` is `school_district_name` -> `school_district_id` and `creator_email` -> `creator_id`. But, you're actually returning the district name and creator email values (which is what I'd expect the FE would want). For example, the following test assertions:

```python
    assert data[0].get("school_district_id") == school_district_name
    assert data[0].get("creator_id") == "manager@rice.edu"
```


We're clearly returning name / email, but calling the fields `_id` :confused: . It seems to me these should be named `school_district_name` and `creator_email` respectively, in which case we can use the existing `FullCourseBuildSettings` model :man_shrugging: .

Reply:
Let me look at this one again. I may have gotten the schema requirements mixed up with the response model.